server/service/integration_core_test.go

```diff
@@ -2662,41 +2663,62 @@ func (s *integrationTestSuite) TestVulnerabilitiesWebhookConfig() {
 	require.Equal(t, 1*time.Hour, config.WebhookSettings.Interval.Duration)
 }
 
-func (s *integrationTestSuite) TestIntegrationsConfig() {
+func (s *integrationTestSuite) TestExternalIntegrationsConfig() {
 	t := s.T()
 
-	// create a test http server to act as the Jira server
+	// create a test http server to act as the Jira and Zendesk server
 	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if r.Method != "GET" {
 			w.WriteHeader(501)
 			return
 		}
-		if r.URL.Path != "/rest/api/2/project/qux" {
-			w.WriteHeader(502)
-			return
-		}
+		if r.URL.Path == "/rest/api/2/project/qux" {
```

Comment:
Nit: `switch r.URL.Path {` (instead of the `else if` chains)

Reply:
Thank you! I talked myself out of doing this the first time around thinking that maybe folks might find the nested switches less readable. But I like this approach better :)